### PR TITLE
add simple galaxy tutorial

### PR DESCRIPTION
**Pull request by ismael-mendoza**
Hi @aboucaud - here is what I had in mind for the small galaxy guide. Please let me know if you have any suggestions :)

### CODE REVIEW

File: docs/guides/create-galaxy.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Create Simple Galaxy with Galcheat
```
Comment (aboucaud) [inline docs/guides/create-galaxy.md:1]:
```suggestion
# Simulate a simple galaxy with galcheat and GalSim
```

File: docs/guides/create-galaxy.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+In this tutorial, we will see how we can use the survey parameters in galcheat
+to create a simple galsim elliptical galaxy. Our galaxy will be convolved with
```
Comment (aboucaud) [inline docs/guides/create-galaxy.md:4]:
```suggestion
to create a simple elliptical galaxy with galsim. The galaxy will be convolved with
```

**Review by aboucaud — COMMENTED** [review]
Looks good, thanks Ismael.

Suggestion : maybe you could separate the tutorial in several code blocks to ease the reading.

1. create a galaxy model and shear it
2. create an atmospheric PSF
3. create an optical PSF
4. convolve with the model
5. add noise

**Comment by aboucaud** [comment]
Closes #61

File: docs/guides/create-galaxy.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+a optical+atmospheric component PSF, include background and noise, and use
+the r-band filter of the LSST survey.
+
+```python
+import galsim
+from galcheat import available_surveys
+from galcheat import get_survey, get_filter
```
Comment (aboucaud) [inline docs/guides/create-galaxy.md:11]:
```suggestion
from galcheat import get_survey
```

`get_filter` is deprecated

File: docs/guides/create-galaxy.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+from galcheat.utilites import mag2counts, mean_sky_level
+
+LSST = get_survey("LSST")
+r_band = get_filter("r", "LSST")
```
Comment (aboucaud) [inline docs/guides/create-galaxy.md:15]:
```suggestion
r_band = LSST.get_filter("r")
```

File: docs/guides/create-galaxy.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
+# Create Simple Galaxy with Galcheat
+
+In this tutorial, we will see how we can use the survey parameters in galcheat
+to create a simple galsim elliptical galaxy. Our galaxy will be convolved with
+a optical+atmospheric component PSF, include background and noise, and use
+the r-band filter of the LSST survey.
+
+```python
+import galsim
+from galcheat import available_surveys
```
Comment (aboucaud) [inline docs/guides/create-galaxy.md:10]: unused

File: docs/guides/create-galaxy.md (hunk 3 above)
Comment (ismael-mendoza) [reply docs/guides/create-galaxy.md:11]: thanks for the catch!

**Comment by ismael-mendoza** [comment]
Thanks @aboucaud for your helpful suggestions :) Please let me know if there is anything else, I just updated it

**Comment by ismael-mendoza** [comment]
Maybe a link to tutorials or just this one in the README might be good ?

**Review by aboucaud — APPROVED** [review]
Nice !

After your last sentence, I think I just want to see the kind of image produced.

Would you be able to produce one with the script and add it to the docs ?

**Comment by ismael-mendoza** [comment]
Yup, one sec

**Comment by ismael-mendoza** [comment]
done : )

**Pull request merged**